frontend/containers/InteractiveOverviewPage/InteractiveOverviewPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import Card from "@/components/Card/Card";
+import { basePageWrap } from "@/containers/BasePage";
+import HeroBlock from "@/components/Blocks/HeroBlock/HeroBlock";
+import HeaderFullImageBlock from "@/components/Blocks/HeaderFullImageBlock/HeaderFullImageBlock";
+
+const InteractiveOverviewPage = ({
+  hero,
+  overviewPages,
+}: {
+  hero: any[];
+  overviewPages: any[];
+}) => {
+  return (
+    <React.Fragment>
+      {hero?.map(contentItem => {
+        switch (contentItem.type) {
+          case "header_full_image_block":
+            return <HeaderFullImageBlock key={`headerfull ${contentItem.id}`} data={contentItem} />;
+          case "hero_block":
+            return <HeroBlock key={`heroblock ${contentItem.id}`} data={contentItem} />;
+          default:
+            null;
+        }
```

Comment:
Ook dit kan misschien samengevoegd worden of vervangen worden door een generiek herocomponent